### ISSUE TITLE
LiDAR filtering by quality filter and vertical windspeed threshold

### ISSUE THREAD

**Issue by ShaneBrightWind** [brightwind-dev/brightwind#439]
Create a group of functions that carry out the following functionality:

1. Clean all values out of a LiDAR dataset below a certain quality threshold.
2. Clean out all values outside of a certain +/- vertical windspeed threshold.

**Comment by BiancaMorandi** [comment]
Should this function be a wrap around function calling BrightHub cleaning rule function? e.g only cleaning for windcube, ZX etc...
pull cleaning rules from BH and apply those.

**Comment by ShaneBrightWind** [comment]
@BiancaMorandi I agree that the BH cleaning rule function would capture it for data downloaded from BH. Howver the BW library also allows you to load in raw data files, and work with them directly, so i still think it is valid to have these functions as a separate option to select in the BW library.
